Essentials/src/messages_de.properties

```diff
@@ -127,18 +127,18 @@ extinguishOthers=\u00a76Du hast {0}\u00a76 gel\u00f6scht.
 failedToCloseConfig=Fehler beim Schlie\u00dfen der Konfiguration {0}.
 failedToCreateConfig=Fehler beim Erstellen der Konfiguration {0}.
 failedToWriteConfig=Fehler beim Schreiben der Konfiguration {0}.
-false=\u00a74falsch\u00a7r
+false=\u00a74Nein\u00a7r
```

Comment:
Is this actually more appropriate than `falsch`?

Reply:
in situation like /whois 
yes a lot :)